diff --git a/examples/encoder_example.c b/examples/encoder_example.c
index 4b73b64..7dcb933 100644
--- a/examples/encoder_example.c
+++ b/examples/encoder_example.c
@@ -52,7 +52,7 @@
 #include <fcntl.h>
 #include <io.h>
 
-static double rint(double x)
+static double rint_(double x)
 {
   if (x < 0.0)
     return (double)(int)(x - 0.5);
diff --git a/win32/VS2008/dump_video/dump_video_dynamic.vcproj b/win32/VS2008/dump_video/dump_video_dynamic.vcproj
index 9a6f49f..e1f081e 100644
--- a/win32/VS2008/dump_video/dump_video_dynamic.vcproj
+++ b/win32/VS2008/dump_video/dump_video_dynamic.vcproj
@@ -17,7 +17,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -36,7 +36,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -55,7 +55,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -74,7 +74,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -93,7 +93,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -112,7 +112,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -131,7 +131,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -150,7 +150,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
diff --git a/win32/VS2008/dump_video/dump_video_static.vcproj b/win32/VS2008/dump_video/dump_video_static.vcproj
index 60d6d74..243bc1a 100644
--- a/win32/VS2008/dump_video/dump_video_static.vcproj
+++ b/win32/VS2008/dump_video/dump_video_static.vcproj
@@ -17,7 +17,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -36,7 +36,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -55,7 +55,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -74,7 +74,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -93,7 +93,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -112,7 +112,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -131,7 +131,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -150,7 +150,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
diff --git a/win32/VS2008/encoder_example/encoder_example_dynamic.vcproj b/win32/VS2008/encoder_example/encoder_example_dynamic.vcproj
index d68d8ac..7c63063 100644
--- a/win32/VS2008/encoder_example/encoder_example_dynamic.vcproj
+++ b/win32/VS2008/encoder_example/encoder_example_dynamic.vcproj
@@ -17,7 +17,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -36,7 +36,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -55,7 +55,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -74,7 +74,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -93,7 +93,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -112,7 +112,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -131,7 +131,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -150,7 +150,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib libvorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
diff --git a/win32/VS2008/encoder_example/encoder_example_static.vcproj b/win32/VS2008/encoder_example/encoder_example_static.vcproj
index 91cee2d..2068b37 100644
--- a/win32/VS2008/encoder_example/encoder_example_static.vcproj
+++ b/win32/VS2008/encoder_example/encoder_example_static.vcproj
@@ -17,7 +17,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -36,7 +36,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="2" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="true" SubSystem="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -55,7 +55,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -74,7 +74,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -93,7 +93,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -112,7 +112,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -131,7 +131,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="1"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -150,7 +150,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg_static.lib libvorbis_static.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib vorbisfile.lib vorbisenc.lib vorbis.lib libtheora_static.lib" LinkIncremental="1" AdditionalLibraryDirectories="&quot;..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\..\..\..\libvorbis\win32\VS2008\$(PlatformName)\$(ConfigurationName)&quot;;&quot;..\$(PlatformName)\$(ConfigurationName)&quot;" GenerateDebugInformation="false" SubSystem="1" OptimizeReferences="2" EnableCOMDATFolding="2" OptimizeForWindows98="1" RandomizedBaseAddress="1" DataExecutionPrevention="0" TargetMachine="17"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
diff --git a/win32/VS2008/libtheora/libtheora_dynamic.vcproj b/win32/VS2008/libtheora/libtheora_dynamic.vcproj
index 56fcee5..a6c9325 100644
--- a/win32/VS2008/libtheora/libtheora_dynamic.vcproj
+++ b/win32/VS2008/libtheora/libtheora_dynamic.vcproj
@@ -20,7 +20,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -39,7 +39,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -58,7 +58,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -77,7 +77,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -96,7 +96,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -115,7 +115,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -134,7 +134,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -153,7 +153,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
+			<Tool Name="VCLinkerTool" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\..\lib\theora.def " RandomizedBaseAddress="1" DataExecutionPrevention="0"/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCManifestTool"/>
 			<Tool Name="VCXDCMakeTool"/>
@@ -172,7 +172,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -191,7 +191,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -210,7 +210,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -229,7 +229,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -248,7 +248,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -267,7 +267,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -286,7 +286,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -305,7 +305,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -324,7 +324,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -343,7 +343,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -362,7 +362,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
@@ -381,7 +381,7 @@
 			<Tool Name="VCManagedResourceCompilerTool"/>
 			<Tool Name="VCResourceCompilerTool"/>
 			<Tool Name="VCPreLinkEventTool"/>
-			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="libogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
+			<Tool Name="VCLinkerTool" AdditionalOptions="" AdditionalDependencies="ogg.lib" AdditionalLibraryDirectories="..\..\..\..\libogg\win32\VS2008\$(PlatformName)\$(ConfigurationName)" ModuleDefinitionFile="..\..\libtheora.def "/>
 			<Tool Name="VCALinkTool"/>
 			<Tool Name="VCXDCMakeTool"/>
 			<Tool Name="VCBscMakeTool"/>
diff --git a/lib/theora.def b/lib/theora.def
new file mode 100644
index 0000000..22e4d65
--- /dev/null
+++ b/lib/theora.def
@@ -0,0 +1,56 @@
+LIBRARY	libtheora
+EXPORTS
+	theora_version_string
+	theora_version_number
+	theora_encode_init
+	theora_encode_YUVin
+	theora_encode_packetout
+	theora_encode_header
+	theora_encode_comment
+	theora_encode_tables
+	theora_decode_header
+	theora_decode_init
+	theora_decode_packetin
+	theora_decode_YUVout
+	theora_control
+	theora_packet_isheader
+	theora_packet_iskeyframe
+	theora_granule_shift
+	theora_granule_frame
+	theora_granule_time
+	theora_info_init
+	theora_info_clear
+	theora_clear
+	theora_comment_init
+	theora_comment_add
+	theora_comment_add_tag
+	theora_comment_query
+	theora_comment_query_count
+	theora_comment_clear
+	th_version_string
+	th_version_number
+	th_decode_headerin
+	th_decode_alloc
+	th_setup_free
+	th_decode_ctl
+	th_decode_packetin
+	th_decode_ycbcr_out
+	th_decode_free
+	th_packet_isheader
+	th_packet_iskeyframe
+	th_granule_frame
+	th_granule_time
+	th_info_init
+	th_info_clear
+	th_comment_init
+	th_comment_add
+	th_comment_add_tag
+	th_comment_query
+	th_comment_query_count
+	th_comment_clear
+	th_encode_alloc
+	th_encode_ctl
+	th_encode_flushheader
+	th_encode_packetout
+	th_encode_ycbcr_in
+	th_encode_free
